inc/healthcheck/namespace.php

```diff
@@ -60,7 +60,7 @@ function output_page( array $checks ) {
 	if ( ! empty( $_SERVER['HTTP_ACCEPT'] ) && $_SERVER['HTTP_ACCEPT'] === 'application/json' ) {
 		$format = 'json';
 	}
-	if ( $_GET['_accept'] ?? '' === 'json' ) {
+	if ( isset( $_GET['_accept'] ) && $_GET['_accept'] === 'json' ) {
```

Comment:
Why is this changed?

Reply:
Was getting some linter errors but I can change it back